zenoh-plugin-ros2dds/src/ros2_utils.rs

```diff
@@ -393,6 +418,30 @@ pub fn check_ros_name(name: &str) -> Result<(), String> {
     }
 }
 
+pub fn extract_type_hash(qos: &Qos) -> Option<String> {
```

Comment:
The compiler reports this function is never used:

warning: function `extract_type_hash` is never used
   --> zenoh-plugin-ros2dds/src/ros2_utils.rs:421:8
    |
421 | pub fn extract_type_hash(qos: &Qos) -> Option<String> {
    |        ^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(dead_code)]` on by default

warning: `zenoh-plugin-ros2dds` (lib) generated 1 warning